scripts/git_divergence/divergence.py

```diff
@@ -2,14 +2,25 @@
 import os
 import re
 import sys
+import argparse
 
 from git import Repo
 
 MASTER = os.getenv("GIT_AUTH_BRANCH", default="master")
 
 DATETIME = "%Y-%m-%d %H:%M:%S%z"
 
-BRANCH_FILTER = r"(.*RC\/.*)|([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)"
+ap = argparse.ArgumentParser()
```

Comment:
Nice! I think something like `argparse` makes sense to add here :+1: 

But one thing thats immediately noticeable to me is: we should make it consistent for both `GIT_AUTH_BRANCH` and the `BRANCH_FILTER` to be environment variables as well as arguments. I think `argparse` does this OK but we might  want to take a look at click: https://click.palletsprojects.com/en/8.0.x/why/#why-click since were introducing cli flags